list_to_email_converter.py

```diff
@@ -0,0 +1,215 @@
+                akt_lines = akt_lines+1
+                if (line ==''):
+                    email.close()
+                    email = open('_data/people.yml', 'r')
+                if line.find(people[i]+":")!= -1:
+                    helpvar = 1
+                elif helpvar > 0 and line.find('  sur_name:') != -1:
+                    name = name + line.replace("  sur_name: ", "").rstrip()
+                    helpvar = helpvar + 1
+                elif helpvar > 0 and line.find('  given_name:') != -1:
+                    name = line.replace("  given_name: ", "").rstrip() + " " + name
+                    helpvar = helpvar + 1
+                elif helpvar > 0 and line.find('  email:') != -1:
+                    if(line == "  email:\n" or line == "  email: \n"):
+                        print(name +" has no Emailaddress.")
+                        emailaddress = "!!! "+ name + " HAS NO EMAILADDRESS!!!"
+                    else:
+                        emailaddress = line.replace("  email: ", "").rstrip()
+                    helpvar = helpvar + 1
+                elif helpvar == 4:
+                    name = name.replace("  "," ")
+                    break
+                if (akt_lines >= num_lines):
+                    email.seek(0)
+                    akt_lines = 0
+
+            head.append([people[i], name, emailaddress])
+            realname[people[i]] = name
+            i=i+1
+            ## checks email and real name of the projecleaders
+
+        else:
+            while people[i] == people[i + 1]:
+                people.pop(i+1)
+            if people[i+1].find('*') != -1 and people[i] == people[i+1].replace("*", ""):
+                i=i+1
+                continue
+                ## if one person is projectleader
+            helpvar = 0
+            name = ""
+            for line in email:
+                akt_lines = akt_lines + 1
+                if (line == ''):
+                    email.close()
+                    email = open('_data/people.yml', 'r')
+                if line.find(people[i]) != -1:
+                    helpvar = 1
+                elif helpvar > 0 and line.find('  sur_name:') != -1:
+                    name = name + line.replace("  sur_name:", "").rstrip()
+                    helpvar = helpvar + 1
+                elif helpvar > 0 and line.find('  given_name:') != -1:
+                    name = line.replace("  given_name: ", "").rstrip() + " " + name
+                    helpvar = helpvar + 1
+                elif helpvar == 3:
+                    name = name.replace("  ", " ")
+                    break
+                if (akt_lines >= num_lines):
+                    email.seek(0)
+                    akt_lines = 0
+            realname[people[i]]= name
+            i=i+1
+            ## checks the real names of the persons
+
+    #print (people)
+    #print (head)
+    #print (realname)
+
+    email.close()
+
+    datei = open("projectleader_mailinglist.txt", "w")
```

Comment:
You could add a flag like `-mailinglist` with argparse. Only if this is set, create the mailinglist, otherwise skip this step. In particular if only a single file is passed as parameter, we may not need this mailinglist and it'd be good to do this only on demand.